requirements.txt

```diff
@@ -1,9 +1,12 @@
 # setup.py install_requires
 # any changes here should also be reflected in setup.py "install_requires"
-s3fs >= 2021.7.0
-tenacity >= 8.0.1
-ray == 1.10.0
-pandas >= 1.3.1
-pyarrow >= 5.0.0
-numpy >= 1.21.1
-boto3 == 1.20.40
+"s3fs == 2022.1.0",
```

Comment:
Can we make sure we include all of the changes from https://github.com/ray-project/deltacat/commit/40900c81df315280e3e1cceecc51886677ec9fab here and in setup.py (we can also set the mainline version to 0.2.0 in __init__.py for now).

Reply:
sure